opendbc/safety/modes/rivian.h

```diff
@@ -3,22 +3,13 @@
 #include "opendbc/safety/declarations.h"
 
 static uint8_t rivian_get_counter(const CANPacket_t *msg) {
-  uint8_t cnt = 0;
-  if ((msg->addr == 0x208U) || (msg->addr == 0x150U)) {
-    // Signal: ESP_Status_Counter, VDM_PropStatus_Counter
-    cnt = msg->data[1] & 0xFU;
-  }
-  return cnt;
+  // Signal: ESP_Status_Counter, VDM_PropStatus_Counter
+  return msg->data[1] & 0xFU;
 }
```

Comment:
I don't necessarily like that this test is changing our style, but in this case, I think it's a net win and increases readability.